lib/src/truecaller.dart

```diff
@@ -30,6 +30,7 @@
 
 import 'dart:collection';
 import 'dart:convert';
+import 'dart:io';
```

Comment:
io seems like an unused import, can be removed.

Reply:
Done